docs-api/frontity-packages/features-packages/smart-ads.md

```diff
@@ -0,0 +1,200 @@
+---
+description: API reference of `@frontity/smart-adserver` package
+---
+
+# @frontity/smart-adserver
+
+Smart Adserver is an ad serving network and they provide an API for displaying the ads which is described [here](https://support.smartadserver.com/s/article/Tagging-guide)
+
+The `@frontity/smart-adserver` package will load a third-party Smart Adserver library which adds certain properties on the global `window` object. Then you as the developer can then use those properties to make the "ad call" (which are basically API calls to the Smart Adserver).
+
+In response, the Smart Adserver dynamically loads some code which will modify the DOM to insert the ad in the place that was specified in the API call.
```

Comment:
Uhm, this is not completely rght. It's true that `@frontity/smart-adserver` loads a third-party library, but developers wouldn't have to interact with it. The one that uses the SmartAdserver API is the `<SmartAd>` component exposed in `libraries.fills.smartAdserver.SmartAd`.

I would write something like this instead:

```suggestion
The `@frontity/smart-adserver` package will load a third-party Smart Adserver library which adds certain properties on the global `window` object. Then you as the developer can render the exposed `<SmartAd>` component, which will use those properties to make the "ad call" (which are basically API calls to the Smart Adserver).

In response, the Smart Adserver dynamically loads some code which will modify the DOM to insert the ad in the place that the `<SmartAd>` component was rendered.
```